pkg/api/v20230401/openshiftcluster.go

```diff
@@ -126,8 +126,10 @@ type ServicePrincipalProfile struct {
 	ClientSecret string `json:"clientSecret,omitempty" mutable:"true"`
 }
 
+// OutboundType represents the type of routing a cluster is using.
 type OutboundType string
 
+// OutboundType constants.
```

Comment:
Hey Carlo, thanks for your work on this. Does this need to be backported to any of the other pkg/api versioned directories?